dist/profile/templates/azure.ci.jenkins.io/agents.yaml.erb

```diff
@@ -0,0 +1,502 @@
+---
+jenkins:
+  ## No builds on master
+  numExecutors: 0
+  agentProtocols:
+  - "JNLP4-connect"
+  - "Ping"
```

Comment:
This is the default IIRC, you should not need to override it.

Reply:
I've put them here for the sake of knowledge sharing, to make sure that the reader/contributor knows it